File: 242-ValidAnagram/242-ValidAnagram.java
// Last updated: 7/14/2026, 2:03:38 PM
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length()!=t.length()){
            return false;
        }
        int []arr=new int[26];
        for(char ch:s.toCharArray()){
            arr[ch-'a']++;
        }
        for(char c:t.toCharArray()){
            if(arr[c-'a']==0){
                return false;
            }arr[c-'a']--;
        }return true;

    }
}

File: 242-ValidAnagram/242-ValidAnagram.py
# Last updated: 7/14/2026, 2:03:32 PM
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s)!=len(t):
            return False
        ss=sorted(s)
        ts=sorted(t)
        for i,j in zip(ss,ts):
            if i!=j:
                return False
        return True
            
        